src/ast/mod.rs

```diff
@@ -99,7 +100,30 @@ where
             GenericNCommand::Sort(span, name, params) => {
                 GenericCommand::Sort(span.clone(), *name, params.clone())
             }
-            GenericNCommand::Function(f) => GenericCommand::Function(f.clone()),
+            // This is awkward for the three subtypes change
```

Comment:
is this "change" a todo?

Reply:
The case here is that this change broke the one-to-one correspondence between `GenericCommand` and `GenericNCommand`. I'm not sure what to do with that.

Reply:
I think it's okay, just wondering what you meant here